Core/Base/AgentModule.py

```diff
@@ -20,13 +18,16 @@
 from DIRAC.Core.Utilities.ReturnValues import isReturnStructure
 from DIRAC.Core.Utilities import Time, MemStat
 
+__RCSID__ = "$Id$"
+
+
 def _checkDir( path ):
   try:
     os.makedirs( path )
-  except Exception:
+  except OSError:
```

Comment:
I think to remember you can also get IOError exceptions.
```except ( OSError, IOError ) ``` ? 
Also, I don't really understand: the OS exception is caught, to be raised again ? 

```
def _checkDir( path ):
  os.makedirs( path )
```
Would do the same no ?